shared/haml/course_explorer_table.haml

```diff
@@ -20,11 +20,141 @@
-- courses << { name: "CS Principles", starts: 9, ends: 12, subtitle: "Full year course", link: CDO.code_org_url("/educate/csp"), img: "/shared/images/teacher-announcement/csp-widgets.png", description: "In fall 2016, the College Board launched its newest AP® course, AP Computer Science Principles. The course introduces students to the foundational concepts of computer science and challenges them to explore how computing and technology can impact the world. The AP Program designed AP Computer Science Principles with the goal of creating leaders in computer science fields and attracting and engaging those who are traditionally underrepresented with essential computing tools and multidisciplinary opportunities. The curriculum is flexible to be taught as a normal course or as an AP course.", prereqs: "No prerequisites needed. This course works for beginners or students with experience in CS Fundamentals or CS Discoveries."}
+:ruby
+  courses = []
+  courses << {
+    name: "CS Principles",
+    starts: 9,
+    ends: 12,
+    subtitle: "Full year course",
+    link: CDO.code_org_url("/educate/csp"),
+    course_link: CDO.studio_url("/courses/csp"),
+    img: "/shared/images/teacher-announcement/csp-widgets.png",
+    description: 'AP Computer Science Principles is a course designed to prepare students (and teachers) who are new to computer science for the AP CS Principles exam. The course covers many topics including the Internet, Big Data and Privacy, and Programming and Algorithms.
+      <ul>
+        <li> <strong>Length:</strong> Full-year course (120-180 hours) </li>
+        <li> <strong>Recommended age:</strong> Ages 13-18 (9th through 12 grade) </li>
+        <li> <strong>Prerequisites:</strong> None </li>
+        <li> <strong>Optional professional learning:</strong> Free 1-week summer workshop, three 1-day mid-year workshops, and online support. </li>
+        <li> <strong>Cost to use Curriculum:</strong> None </li>
+        <li> <strong>Languages:</strong> English only </li>
+      </ul>'
+  }
+
+  courses << {
+    name: "CS Discoveries",
+    starts: 7,
+    ends: 9,
+    subtitle: "Semester or full year course",
+    link: CDO.code_org_url("/educate/csd"),
+    course_link: CDO.studio_url("/courses/csd"),
+    img: "/shared/images/fit-500/courses/logo_csd.jpg",
+    description: 'CS Discoveries is an introductory course that empowers students to engage with Computer Science as a medium for creativity, communication, problem solving, and fun.
+      <ul>
+        <li> <strong>Length:</strong> Semester or full-year course (? hours) </li>
+        <li> <strong>Recommended age:</strong> Ages 11-14 (6th through 9th grade) </li>
+        <li> <strong>Prerequisites:</strong> None </li>
+        <li> <strong>Optional professional learning:</strong> Free 1-day workshop, in person throughout the US and   line globally. </li>
```

Comment:
Free 1-week summer workshop, four 1-day workshops throughout the academic year, and online support. [Learn more here](https://code.org/educate/professional-learning/cs-discoveries).